makefile

```diff
@@ -332,10 +332,16 @@ $(EXECUTABLE): $(CNFS_FILE) $(OBJECTS)
 
 # To create the c file with assets, run these tools
 $(CNFS_FILE):
+# Sokoban .tmx to bin preprocessor
+	python ./tools/soko/soko_tmx_preprocessor.py ./assets/soko/ ./assets_image/
```

Comment:
Python is already a requirement for the dev env, and it doesnt look like you're using any nonstandard packages. Should be fine.